CreateAnsSaveFiles.py

```diff
@@ -0,0 +1,34 @@
+import os
+import shutil
+
+
+class CreateAndSaveFiles:
+
+    def __init__(self, new_folder_path, folder_name, new_full_names, ext_dir_of_photos, path):
+        self.new_folder_path = new_folder_path
+        self.folder_name = folder_name
+        self.new_full_names = new_full_names
+        self.ext_dir_of_photos = ext_dir_of_photos
+        self.path = path
+
+    def create_new_folder(self):
+        self.new_folder_dir = os.path.join(
+            self.new_folder_path, self.folder_name)
+        if not os.path.exists(self.new_folder_dir):
+            os.makedirs(self.new_folder_dir)
+            print('New folder has been created.')
+            print('-'*70)
+        else:
+            print('File exists!')
+
+    def save_copy_photos(self):
```

Comment:
Class or method should be doing one thing only. This class creates dictionary and copies files there - should be split. Good indication  of such situation is when you want to create name for class/method and starts to come up with names consisting words such as 'and', 'or'. 

For further lecture see Single Responsibility Principle